packages/nativescript-in-app-review/index.android.ts

```diff
@@ -0,0 +1,33 @@
+import { Application, Utils } from '@nativescript/core';
+import { InAppReviewCommon } from './common';
+
+export class InAppReview extends InAppReviewCommon {
+  static requestReviewPopup() {
+    const context = Utils.android.getApplicationContext() as android.content.Context;
+    const manager = com.google.android.play.core.review.ReviewManagerFactory.create(context);
+    const request = manager.requestReviewFlow();
+
+    request.addOnCompleteListener(
+      new com.google.android.play.core.tasks.OnCompleteListener({
+        onComplete: (task) => {
+          if (!task.isSuccessful()) {
+            return;
+          }
+
+          const reviewInfo = task.getResult();
+          const flow = manager.launchReviewFlow(Application.android.foregroundActivity, reviewInfo);
+
+          flow.addOnCompleteListener(
+            new com.google.android.play.core.tasks.OnCompleteListener({
+              onComplete: () => {
+                // The flow has finished. The API does not indicate whether the
```

Comment:
also the promise might complete at this point (in which case we know that the user has finished the flow). Which might be useful for something like:

```
await InAppReview. requestReviewPopup();
// navigate away
```